packages/design-system-react-native/jest.config.js

```diff
@@ -23,23 +23,17 @@ module.exports = merge(baseConfig, {
       statements: 100,
     },
   },
-  collectCoverageFrom: [
-    '!**/*.scripts.{js,ts}', // Exclude .scripts files
-    '!**/node_modules/**', // Exclude node_modules
-    '!**/dist/**', // Exclude build outputs
-    '!**/*.constants.{js,ts}', // Exclude .constants files
-    '!**/*.dev.{js,ts}', // Exclude .dev files
-    '!**/*.assets.{js,ts}', // Exclude .assets files
-    '!**/*.types.{js,ts}', // Exclude .types files
-    '!./src/components/temp-components/Blockies/Blockies.utilities.ts',
-  ],
+
```

Comment:
Exclusions for .scripts, .constants, .dev, .assets, .types files, as well as specific files like Blockies.utilities.ts are more appropriate in the coverage ignore section rather than in the collectCoverageFrom array.

- `collectCoverageFrom` is intended to specify which files Jest should collect coverage information from. While it does support exclusion patterns prefixed with `!`, it’s more idiomatic and maintainable to use the `coveragePathIgnorePatterns` array for exclusions.